examples/2fa-sms-login.example.ts

```diff
@@ -0,0 +1,47 @@
+import { IgApiClient } from '../src';
+import { get } from 'lodash';
+
+const readline = require('readline');
+const rl = readline.createInterface({ input: process.stdin, output: process.stdout });
+
+(async () => {
+  // Initiate Instagram API client
+  const ig = new IgApiClient();
+  ig.state.generateDevice(process.env.IG_USERNAME);
+  ig.state.proxyUrl = process.env.IG_PROXY;
+
+  let loggedInUser;
+  try {
+    // Perform usual login
+    loggedInUser = await ig.account.login(process.env.IG_USERNAME, process.env.IG_PASSWORD);
+  } catch (e) {
```

Comment:
You might want to use Bluebird like here: https://github.com/dilame/instagram-private-api/blob/a716332eb91c6a6986af058f0016b01aec4cbc91/examples/checkpoint.example.ts#L13

Reply:
No problem, just wanted to use as fewer dependencies as possible

Reply:
This library already depends on Bluebird so this should not be an issue

Reply:
I'd suggest moving onto Bluebird promises completely then. Or, as an alternative, adding a possibility to pick Promise implementation when the library is getting imported. What do you think? @Nerixyz

Reply:
For what reason would you like to change the promise library?